prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctionsTests.kt

```diff
@@ -323,4 +336,137 @@ class CustomFHIRFunctionsTests {
             )
         }.isFailure()
     }
+
+    @Test
+    fun `test changeTimezone`() {
+        val isoFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssXXX")
```

Comment:
Add tests with partial date times:

1. What happens if there is only a year?
2. What happens if there is only a date (1/1/2023)?
3. Make a test for Instant type too.
Then figure out if that behavior is correct.